front/app/components/UI/Image/index.tsx

```diff
@@ -96,5 +103,11 @@ export default class Image extends PureComponent<Props, State> {
         loading={isLazy ? 'lazy' : 'eager'}
       />
     );
+
+    if (cover) {
+      image = <Fallback src={src} className={className} />;
+    }
```

Comment:
So this "Fallback" is used whenever the cover is set to true.. so it's not really a Fallback right? Or am I misunderstanding something?

Reply:
Indeed not a fallback then, but maybe only needed in this place and not for all the other instances where we set cover to true (such as project cards). (And I should have done better testing.)

Reply:
I think we still want to remove this fallback but needs a bit more time to check everything properly.

Reply:
I'll add it to our backlog for cooldown weeks.

Reply:
Ok sounds good! I believe it's only the 2-column banner which had a visible impact anyways, so it would have been tricky to catch in any case :slightly_smiling_face: